docs/UserGuide/6-System Tools/3-JMX Tool.md

```diff
@@ -23,6 +23,30 @@
 
 Java VisualVM is a tool that provides a visual interface for viewing detailed information about Java applications while they are running on a Java Virtual Machine (JVM), and for troubleshooting and profiling these applications. 
 
+## Config
+
+If you have set access and password for JMX (that is, `com.sun.management.jmxremote.authenticate` is true), you need to edit user config in `iotdb-engine.properties`. Related configurations are:
```

Comment:
Hi,
we need to set this as true by default. 
please modify the iotdb-env.sh script.

Reply:
Modified. Please re-review : )